src/test/lambda/models/samLambdaRuntime.test.ts

```diff
@@ -123,3 +124,20 @@ describe('runtimes', function () {
         ])
     })
 })
+
+describe('getNodeMajorVersion()', () => {
+    it('should extract major version from nodejs12.x', () => {
+        const version = getNodeMajorVersion('nodejs12.x')
+        assert.strictEqual(version, 12)
+    })
+
+    it('should extract major version from nodejs18.x', () => {
+        const version = getNodeMajorVersion('nodejs18.x')
+        assert.strictEqual(version, 18)
+    })
+
+    it('should return undefined on invalid input', () => {
```

Comment:
"should" is always redundant in test descriptions. 
```suggestion
    it('returns undefined on invalid input', () => {
```